common/commands/config.go

```
@@ -79,25 +80,30 @@ func (cc *ConfigCommand) SetDetails(details *config.ServerDetails) *ConfigComman
 	return cc
 }
 
-func (cc *ConfigCommand) Run() error {
+func (cc *ConfigCommand) Run() (err error) {
 	log.Debug("Locking config file to run config " + cc.cmdType + " command.")
-	defer log.Debug("Config " + cc.cmdType + " command completed successfully. config file is released.")
 	mutex.Lock()
-	defer mutex.Unlock()
+
+	defer func() {
+		mutex.Unlock()
```

Let's not ignore this error